github-api/src/main/java/org/eclipse/egit/github/core/util/FilterConstants.java

```diff
@@ -0,0 +1,21 @@
+package org.eclipse.egit.github.core.util;
+
+public class FilterConstants {
+    public static final String FILTER_ALL = "all";
+    public static final String OWNER = "owner";
+    public static final String MEMBER = "member";
+    public static final String PRIVATE = "private";
+    public static final String PUBLIC = "public";
+    public static final String SOURCES = "sources";
+    public static final String FORKS = "forks";
+    public static final String WATCHED = "watched";
+    public static final String STARRED = "starred";
+    public static final String FULL_NAME = "full_name";
+    public static final String MAIN_TAG = "main";
+    public static final String SEARCH_TAG = "search";
```

Comment:
Fragment tags don't belong into the Github API, but to their respective activity.

All other items are filter fields (right?) and thus should be moved to the respective service (RepositoryService, I guess) with appropriate naming.